.github/workflows/java.yml

```diff
@@ -197,9 +197,8 @@ jobs:
               name: lint java rust
 
     test-modules:
-        if: github.event.pull_request.head.repo.owner.login == 'valkey-io'
-        environment: AWS_ACTIONS
-        name: Running Module Tests
+        if: (github.repository_owner == 'valkey-io' && github.event_name == 'workflow_dispatch') || github.event.pull_request.head.repo.owner.login == 'valkey-io'
```

Comment:
❤️

Reply:
Why removing the environment ? It's kinda mandatory for security reasons 🙁

Reply:
Only chosen can update branches on upstream

Reply:
I know, but still its preferred to use this

Reply:
I agree with Shoham.  It doesn't hurt, and want robust security measures, not closely coupled security measures.  
Let's re-add
```
environment: AWS_ACTIONS
```

Reply:
added